code/modules/mob/living/silicon/mommi/mommi_modules.dm

```diff
@@ -95,3 +95,40 @@
 		"Gravekeeper" = "cogspider"
 		)
 	law_type = "Gravekeeper"
+
+
+/obj/item/weapon/robot_module/mommi/sammi
+	name = "stationary assembly mmi robot module"
+
+/obj/item/weapon/robot_module/mommi/sammi/New(var/mob/living/silicon/robot/R)
+	//..()
+	languages = list(	LANGUAGE_GALACTIC_COMMON = 0, LANGUAGE_TRADEBAND = 0, LANGUAGE_VOX = 0,
+						LANGUAGE_ROOTSPEAK = 0, LANGUAGE_GREY = 0, LANGUAGE_CLATTER = 0,
+						LANGUAGE_MONKEY = 0, LANGUAGE_UNATHI = 0, LANGUAGE_SIIK_TAJR = 0,
+						LANGUAGE_SKRELLIAN = 0, LANGUAGE_GUTTER = 0, LANGUAGE_MONKEY = 0,
+						LANGUAGE_MOUSE = 0, LANGUAGE_HUMAN = 0)
+	add_languages(R)
+	src.emag = new /obj/item/borg/stun(src)
```

Comment:
Everything past this point is handled by another function. We divorced module loading from new a while ago